Developers-Notes/CONP_data_upload_admin.md

```diff
@@ -0,0 +1,45 @@
+
+Log in to ```sftp.conp.ca``` as user ```proftpd```.  If you need the password for this account, contact @cmadjar or @emmetaobrien.
+
+# Setting up an account on the CONP sftp server
+
+## Create user home directory
+
+```
+mkdir /data/proftpd/users/<USERNAME>
+```
+
+## Create virtual user
+
+```
+ftpasswd --passwd --sha512 --file /opt/conp-proftpd/runtime/etc/sftpd.passwd --uid 1001 --gid 1001 --shell /bin/false --home /data/proftpd/users/<USERNAME> --name <USERNAME>
+```
+
+(Everything in this command remains constant except for the USERNAME).
+
+This command will print some warnings, which are not known to be problematic, and then require you to specify an initial password for USERNAME.
+
+## Set permissions for user directory
+
+```
+chmod go= /data/proftpd/users/<USERNAME>
+```
+
+## Add the user to the quota table (for user with default 50GB quota)
```

Comment:
does this mean that this is only for the users that use less than 50GB quota? 

Note: I would tend to move the information in brackets inside the paragraph instead of the title.